yente/data/common.py

```diff
@@ -121,6 +121,11 @@ class EntityExample(BaseModel):
 
 class EntityMatchQuery(BaseModel):
     weights: Dict[str, float] = Field({}, examples=[{"name_literal": 0.8}])
+    config: Dict[str, Union[str, int, float, bool]] = Field(
```

Comment:
Pydantic has this, in case we want to support nested config structures

```suggestion
    config: Dict[str, JsonValue] = Field(
```

Reply:
Oh I see this matches ConfigVar